README.md

```diff
@@ -20,7 +20,7 @@ Requires Python >= 3.7 and PyTorch >= 1.11
 From pip:
 
 ```bash
-pip install torchsnapshot
+pip install --pre torchsnapshot-nightly
```

Comment:
should we document both `pip install torchsnapshot` for the stable versions, and `pip install --pre torchsnapshot-nightly` for the nightlies? otherwise we don't mention the torchsnapshot package explicitly (outside of the badges)

Reply:
I was thinking about that - maybe we can add the stable versions once we do a major release? (i.e. alpha or beta)

Reply:
that sgtm!